ramls/schema/loan-policy-id.json

```diff
@@ -7,6 +7,13 @@
     "loanPolicyId": {
       "description": "ID of the loan policy determined by the circulation rules",
       "type": "string"
+    },
+    "appliedRuleConditions": {
```

Comment:
Similarly to the internal design, I think we need to consider more specifically structuring this.

What are the contents of this array?

Reply:
updated